bpf/include/common.h

```diff
@@ -87,4 +87,150 @@ static inline int kmesh_map_update_elem(void *map, const void *key, const void *
 
 #define GET_SKOPS_LOCAL_PORT(sk_ops) (__u16)((sk_ops)->local_port)
 
-#endif  // _COMMON_H_
+#define MAX_BUF_LEN 100
+#define MAX_IP4_LEN 16
+#define MAX_IP6_LEN 40
+
+#ifndef u8
+#define u8 __u8
```

Comment:
Why not directly include the corresponding header file?

Reply:
It seems couldnot find u8, u16 type def in ebpf enviroment except include vmlinux.h. So back to use __u8, __u16 instead of u8 u16.